tox.ini

```diff
@@ -7,13 +7,15 @@ envlist =
     docs
     py{37,py37}-dj{32}
     py{38,py38,39,py39,310}-dj{32,40,41,main}
+    py311-djmain
```

Comment:
Does Django 3.2 support it / works there?  If so, it should be added to the line above, and otherwise have its own line with the supported versions/combinations.

Reply:
The only Django version that support Python 3.11 is the one in development now (`djmain`).
I have already checked it in the release notes, but you can check it too if you want.

Reply:
I've added also the latest Django version, 4.1.3, released this morning.